lnbits/core/services.py

```diff
@@ -434,7 +434,7 @@ async def check_admin_settings():
 
         http = "https" if settings.lnbits_force_https else "http"
```

Comment:
you could remove this line because its not needed anymore, i did this so it shows a https url when force_https ENV variable is true.

Reply:
Aye, good point